README.md

```diff
@@ -1,69 +1,46 @@
 # S4S Testsuite
 
-Framework for testing S4S API implementations
+Framework for testing S4S API implementations. The interface is a [Flask](http://flask.pocoo.org/) application, and the tests themselves are run as [Celery](http://www.celeryproject.org/) tasks. The tests are written using the [behave](http://behave.readthedocs.io/en/stable/index.html) framework.
 
-## Build and run in docker
+## S4S reference stack
+
+The test suite is one component of the [Sync for Science reference stack](https://github.com/sync-for-science/reference-stack-docker), which contains additional services such as DSTU2 and STU3 FHIR servers. The `SMART EHR` servers in the vendor list of the test suite refer to the servers that run with the reference stack, so they will be unavailable if you choose to run the test suite independently. *Additionally*, the FHIR resource validation steps will fail when the test suite is run independently, because those FHIR servers are used to perform the validation.
+
+## Build and run in Docker
+
+We use [Docker](https://www.docker.com/) to set up all the required services and run the test server.
 
     git clone https://github.com/sync-for-science/test-suite
     cd test-suite
     docker build -t tests .
     docker run --rm -it \
-      -e BASE_URL=http://tests.dev.syncfor.science:9003 \
-      -p 5000:5000 \
+      -p 9003:5000 \
       tests
-      
-### Develop in docker
 
-    docker run --rm -it \
-      -e BASE_URL=http://tests.dev.syncfor.science:9003 \
-      -p 5000:5000 \
-      -v /host/path/to/test-suite:/usr/src/app \
-      tests \
-      /bin/bash
+`9003` is the default port, but you can configure it to run on other ports - just be sure to add `-e BASE_URL=http://localhost:<port>` to the Docker command as well.
       
-... and from inside docker, run "behave" to execute tests.
+### Develop in Docker
 
-## Installation
+If you want to develop on the app using Docker, mount the repository on the container. Note that the container still has to be restarted to reflect any changes.
```

Comment:
Shouldn't python code changes be picked up automatically by Flask when `FLASK_DEBUG=1`? (If not, we should mention this in the note, specifically that flask reloading won't work.

Reply:
Turns out this does work, but changes to the `features` directory does not trigger a flask reload (which is why I didn't think it was working), since that directory is not part of the flask app. Behave seems to reread the test files every time anyway, so everything should be good for development with `FLASK_DEBUG`.